packages/host/app/routes/card.gts

```diff
@@ -29,16 +32,29 @@ export default class RenderCard extends Route<Model | null> {
     operatorModeEnabled: {
       refreshModel: true,
     },
+    authMode: {
+      refreshModel: true,
+    },
+    sid: {
+      refreshModel: true,
+    },
+    clientSecret: {
+      refreshModel: true,
```

Comment:
Does this need to be `true`?

Reply:
It's needed now so we can remove `sid` and `clientSecret` after user reset password. [9c17253](https://github.com/cardstack/boxel/pull/921/commits/9c1725319658cbd1733310e2e79500bf9cf0a988)